integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/passthru/transport/test/HttpAccessLogTestCase.java

```diff
@@ -135,18 +135,17 @@ private void applyProperty(File srcFile, String key, String value) throws IOExce
             fos = new FileOutputStream(destinationFile);
             Properties properties = new Properties();
             properties.load(fis);
+            fis.close();
             properties.setProperty(key, value);
             properties.store(fos, null);
+            fos.flush();
+            fos.close();
             serverConfigurationManager.applyConfigurationWithoutRestart(destinationFile);
         } catch (Exception e) {
             Assert.assertTrue(false, "Exception occured with the message: " + e.getMessage());
         } finally {
-            if (fis != null) {
-                fis.close();
-            }
-            if (fos != null) {
-                fos.close();
-            }
+            fis = null;
+            fos = null;
```

Comment:
Why do we need to assign null value to fis and fos?

Reply:
Fixed. Please refer latest commit.